util/src/main/java/tc/oc/pgm/util/event/sport/entity/EntityExtinguishEvent.java

```diff
@@ -0,0 +1,24 @@
+package tc.oc.pgm.util.event.sport.entity;
```

Comment:
We don't need the `.sport.` part. But, maybe we should define a `@SportPaper` annotation?